MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -567,16 +570,20 @@ internal void OnPresentationChanged()
 
 #endif
 
-        partial void PlatformValidatePresentationParameters(PresentationParameters presentationParameters)
+        partial void PlatformReset()
         {
+#if WINDOWS
+            CorrectBackBufferSize();
+#endif
+
 #if WINDOWS_UAP
-            if (presentationParameters.SwapChainPanel == null)
+            if (PresentationParameters.SwapChainPanel == null)
                 throw new ArgumentException("PresentationParameters.SwapChainPanel must not be null.");
 #elif WINDOWS_STOREAPP
-            if (presentationParameters.DeviceWindowHandle == IntPtr.Zero && presentationParameters.SwapChainBackgroundPanel == null)
+            if (PresentationParameters.DeviceWindowHandle == IntPtr.Zero && presentationParameters.SwapChainBackgroundPanel == null)
```

Comment:
@cra0zy Missed the lower case `presentationParameters` here

Reply:
ups.